Source/Scene/FeatureMetadata.js

```diff
@@ -0,0 +1,162 @@
+import Check from "../Core/Check.js";
+import defaultValue from "../Core/defaultValue.js";
+import defined from "../Core/defined.js";
+import FeatureTable from "./FeatureTable.js";
+import FeatureTexture from "./FeatureTexture.js";
+
+/**
+ * An object containing feature metadata.
+ * <p>
+ * See the {@link https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0|EXT_feature_metadata Extension} for glTF.
+ * </p>
+ *
+ * @param {Object} options Object with the following properties:
+ * @param {String} options.extension The extension JSON object.
+ * @param {MetadataSchema} options.schema The parsed schema.
+ * @param {Object} [options.bufferViews] An object mapping bufferView IDs to Uint8Array objects.
+ * @param {Object} [options.textures] An object mapping texture IDs to {@link Texture} objects.
```

Comment:
Check throughout the PR for this kind of doc update, I've seen it in a few places

```suggestion
 * @param {Object<String,Uint8Array>} [options.bufferViews] An object mapping bufferView IDs to Uint8Array objects.
 * @param {Object<String,Texture>} [options.textures] An object mapping texture IDs to {@link Texture} objects.
```